docs/install/index.md

```diff
@@ -6,6 +6,11 @@ nav_order: 1
 redirect_from: /docs/how-to/install
 ---
 
+# Install Dependencies before installing ONNX Runtime
```

Comment:
Suggest to update nightly installation commands as in https://github.com/microsoft/onnxruntime/pull/22529/files.